files/default/neutron-ha-tool.py

```diff
@@ -419,7 +425,7 @@ def l3_agent_migrate(qclient, noop=False, now=False):
     LOG.info("%d routers required migration from offline L3 agents",
              total_migrations)
     if total_errors > 0:
-        LOG.error("%d errors encountered during migration" % total_errors)
+        LOG.error("%d errors encountered during migration", total_errors)
```

Comment:
These changes are not related to this commit.  If there is a good reason for them then I'm fine with that, but please put them in a separate commit with a message explaining the change.